components/footer/index.js

```diff
@@ -1,16 +1,24 @@
+import StorageLoss from '../storage-loss'
 import FooterWrapper from './footer-wrapper'
 import InjectionMonth from './injection-month'
 import TimeSlider from './time-slider'
+import useStore from '../../store'
 
 const Footer = () => {
+  const isDOR = useStore((s) => s.isDOR)
   return (
     <>
-      <FooterWrapper bottom={[64, 64, 0, 0]}>
+      <FooterWrapper bottom={isDOR ? [128, 128, 0, 0] : [64, 64, 0, 0]}>
         <InjectionMonth />
       </FooterWrapper>
-      <FooterWrapper>
+      <FooterWrapper bottom={isDOR ? [64, 64, 0, 0] : [0, 0, 0, 0]}>
```

Comment:
It looks like this hard-coded height for mobile/tablet creates this slightly odd effect on mobile/tablet. It's not awful, but it would probably feel better if time slider was still visible when expanding the tooltip on mobile.

![CleanShot 2025-06-03 at 16 13 54@2x](https://github.com/user-attachments/assets/bb7633af-98b2-4eef-aaae-8e92041678e4)